engine/launcher/src/main/scala/org/enso/launcher/cli/LauncherApplication.scala

```diff
@@ -109,6 +112,12 @@ object LauncherApplication {
       "jvm",
       "These parameters will be passed to the launched JVM as -DKEY=VALUE."
     )
+  private def jvmMode =
```

Comment:
- The `jvmMode` flag introduced by this PR is causing some troubles in #13225
- why do we have this flag at all?
- is it needed for something important?
- something that is tested?